src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs

```diff
@@ -3087,37 +3058,24 @@ bool ExpectedErrors(WriteContext writeContext)
                     var simpleCompletion = Assert.IsType<CompletionMessage>(result);
                     Assert.Null(simpleCompletion.Result);
 
-                    // This will log an error on the server as the hub method has completed and will complete all associated streams
+                    // This will log a warning on the server as the hub method has completed and will complete all associated streams
                     await client.SendHubMessageAsync(new StreamItemMessage("id", "error!")).OrTimeout();
```

Comment:
These 2 tests will be updated when https://github.com/aspnet/AspNetCore/issues/7070 is worked on